pypit/arcomb.py

```diff
@@ -3,13 +3,40 @@
 import time
 
 import numpy as np
-from matplotlib import pyplot as plt
 
 from pypit import msgs
 from pypit import arparse as settings
-from pypit import arcycomb
 
-def comb_frames(frames_arr, det, frametype, weights=None, maskvalue=1048577, printtype=None):
+
+def comb_frames(frames_arr, det, frametype, **kwargs):
```

Comment:
At the risk of nagging, I'd like us to avoid using ``**kwargs`` if we can.  I understand why it's there for this case in particular, but to my mind it just obscures what the function does.  For example, the old function could accept `weights=blah`.  This is true of the new function also, but this keyword would never actually get used because core_comb_frames doesn't use it.  Even thought it makes some of the bookkeeping harder on the developers, I think it makes sense to make the code explicit whenever possible.

Reply:
Agreed.  Those kwargs will disappear with the wrapper.